CONTRIBUTING.md

```diff
@@ -1,8 +1,42 @@
 # Contributing to EddieBotUI
+We want to make contributing to this project as easy and transparent as
+possible.
 
-For **help**, **support**, and **questions** please use our [community chat](https://discord.com/invite/jZQs6Wu) 🚑.
+## Our Development Process
+We use GitHub to sync code to and from our internal repository. We'll use GitHub
+to track issues and feature requests, as well as accept pull requests.
+
+## Pull Requests
+We actively welcome your pull requests.
+
+1. Fork the repo and create your branch from `develop`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+
+## Issues
+We use GitHub issues to track bugs. Please ensure your description is
+clear and has sufficient instructions to be able to reproduce the issue.
+
+There are three types of issue templates:
+
+- **Bug report**
+  Use this template if you want to report a bug or a defect. try to be more specific as possible so that everyone can understand and reproduce the bug.
+- **Feature request**
+  If you think that a particular feature should be added.. use this template to explain your idea in details.
+- **Custom issue**
+  Custom template, use this wisely.
+
+## Coding Style  
+* 2 spaces for indentation rather than tabs
+* 80 character line length
+* Run `npm run lint` to conform to our lint rules
+
+## License
+By contributing to Draft.js, you agree that your contributions will be licensed
```

Comment:
I think it should be our repo's name here right 😅 ?
```suggestion
By contributing to EddieBotUI, you agree that your contributions will be licensed
```

Reply:
@BOLT04 yes, I also forgot to mention Draft.js for the template!! I will update the PR tomorrow 💪🏻